index.js

```diff
@@ -200,11 +207,11 @@ module.exports = {
     if (fs.existsSync(dir)) {
       let dirname = path.relative(this.project.root, dir);
       let globs = this.parentRegistry.extensionsForType('js').map((extension) => `**/*.${extension}`);
-
       return walkSync(dir, { directories: false, globs }).map(file => {
         const postfix = hasEmberCliTypescript ? file : file.replace(EXT_RE, '.js');
         const module = prefix + '/' + postfix;
-        this.fileLookup[module] = path.join(dirname, file);
+        let covDirName = dirname.replace(/\.\.\//, '');
+        this.fileLookup[module] = path.join(covDirName, file);
```

Comment:
So the addon figures out that the code in my addons are a directory below (from path.relative). This creates a `../path` for the addons in my monorepo.  Turns out the screws several things up in istanbul, such as where it puts coverage output for addons, which messes links and whatnot up in the reports.  If we strip the `..` it makes instanbul reports much happier (the fileLookup var is used for reports, not coverage calc).